src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        final InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
Wouldn't we want to avoid the round trip of network traffic in SMP and server side processing?  (I know it is not much processing)